cerbero/packages/package.py

```diff
@@ -111,6 +111,8 @@ class PackageBase(object):
     resources_postinstall = 'postinstall'
     resources_postremove = 'postremove'
     conflicts = ''
+    replaces = ''
+    provides = ''
```

Comment:
The new variables should be documented as its done for the rest of variables like in https://github.com/fluendo/cerbero/blob/147c8dbd058dc6836d0d813258edcf7e338a4bdd/cerbero/packages/package.py#L82